### ISSUE TITLE
cannot use HTML provided server-url

### ISSUE THREAD

**Issue by muttleyxd** [mrin9/RapiDoc#94]
I'm providing a custom server-url for my API, yet it isn't setting up properly. This might be related to #64 

```yaml
openapi: 3.0.0
info:
  description: This doesn't work anymore.
  version: 0.0.1
  title: Demo

tags:
  - name: method

paths:
  "/method":
    get:
      operationId: methodGet
      tags:
        - method
      summary: Use get method
      responses:
        200:
          description: successful operation
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/StringResponse"
        500:
          description: server error
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/StringResponse"

servers:
  - url: https://localhost:8080
  - url: http://localhost:8080
components:
  schemas:
    StringResponse:
      type: string
      example: "successful/failed operation"
```

```html
<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <script src="rapidoc-min.js"></script>
  </head>
  <body>
    <rapi-doc server-url="http://192.168.7.32:8080" spec-url="http://192.168.7.32:8080/documentation/documentation.yaml"></rapi-doc>
  </body>
</html>
```

![Screenshot from 2019-11-26 11-06-49](https://user-images.githubusercontent.com/9843181/69620217-467f0380-103d-11ea-906c-fc278756d17d.png)

**Comment by mrin9** [comment]
Thank you for reporting the issue.  I will soon be fixing it. It was introduced in release `7.0.2` where we added the support for `server variables` reported here #88 

Meanwhile, If you arent using `server-variables`   please use the release `7.0.1`

**Comment by mrin9** [comment]
Should be fixed in release `7.0.4`
Feel free to reopen if you still face any issue
